<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="glattingTekst">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Glatter ut og fjerner støy på bildet.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Poisson Bildebehandling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBlur">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Glatting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openContrast">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kontrastforsterkning</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Øker kontrasten i bildet.</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rekonstruerer deler av bilde der informasjon mangler.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openInpainting">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inpainting</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rekonstruerer bildet ut i fra en gråtonemosaikk.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDemosaicing">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Demosaicing</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Limer et bilde sømløst inn i et annet bilde.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openSeamless">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sømløs kloning</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Konverter et bilde til gråtone på to måter.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openGrayConvert">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Konverter til gråtone</string>
    </property>
   </widget>
   <widget class="QLabel" name="glattingTekst_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Oppdager og anonymiserer ansikter på et bilde</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openAnonymise">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Anonymisering</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
